internal/options/options.go

```diff
@@ -30,14 +30,14 @@ func NewCollectorRunOptions() *CollectorRunOptions {
 func (h *CollectorRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(&h.Sources, "source", "source(s) to watch")
 	fs.Var(&h.Sinks, "sink", "external sink(s) that receive data")
-	fs.DurationVar(&h.MetricResolution, "metric_resolution", 60*time.Second, "The resolution at which the collector will collect metrics")
+	fs.DurationVar(&h.PushInterval, "Push_interval", 60*time.Second, "The interval at which the collector will send metrics to the skin")
 	fs.IntVar(&h.MaxProcs, "max_procs", 0, "max number of CPUs that can be used simultaneously. Less than 1 for default (number of cores)")
 	fs.BoolVar(&h.Version, "version", false, "print version info and exit")
 	fs.DurationVar(&h.SinkExportDataTimeout, "sink_export_data_timeout", 20*time.Second, "Timeout for exporting data to a sink")
 	fs.BoolVar(&h.EnableDiscovery, "enable-discovery", true, "enable auto discovery")
 	fs.StringVar(&h.DiscoveryConfigFile, "discovery_config", "", "optional discovery configuration file")
 	fs.StringVar(&h.InternalStatsPrefix, "internal_stats_prefix", "kubernetes.", "optional prefix for internal collector stats")
-	fs.DurationVar(&h.ScrapeTimeout, "scrape_timeout", 20*time.Second, "The per-source scrape timeout")
 	fs.BoolVar(&h.Daemon, "daemon", false, "enable daemon mode")
 	fs.StringVar(&h.ConfigFile, "config_file", "", "required configuration file")
+	fs.IntVar(&h.Verbose, "verbose", 0, "log verbose level")
```

Comment:
Thanks. One of the things we will be doing is moving away from glog/klog. Maybe just retain this verbose flag and use the go log package or logrus.

Also should this be moved into the config instead of as a flag. Will be very useful to simply change the config on the fly to enable logging and then disable again. Rather than having to redeploy if it's a flag.